argopy/errors.py

```diff
@@ -56,4 +56,10 @@ class ErddapServerError(ValueError):
     """
     Raise this when argopy is disrupted by an error due to the Erddap, not argopy machinery
     """
+    pass
+
+class InvalidDashboard(ValueError):
+    """
+    Raise this when trying to work with a 2rd party online service to display float information
```

Comment:
Raise this when trying to work with a 2rd party online service to display float informations